docs/configuration/acme.md

```diff
@@ -553,4 +552,4 @@ For example : if `acme.storage` value is `/etc/traefik/acme/acme.json`, the back
 
 !!! note
     When Træfik is launched in a container, do not forget to create a volume of the parent folder to get the backup file on the host.
-    Otherwise, the backup file will be deleted when the container will be stopped and Træfik will not generate it again.
\ No newline at end of file
+    Otherwise, the backup file will be deleted when the container will be stopped and Træfik will not generate it again.
```

Comment:
Otherwise, the backup file will be permanently deleted when the container is stopped, and Træfik will not be able to generate it again.